resultbrowser/src/js/resultbrowser.js

```diff
@@ -725,154 +861,178 @@
-            .dblclick( function(event) {
-                expandCollapseAction(this.parentNode);
-            });
+            $nameElement.addEventListener(
+                "dblclick",
+                function () {
+                    expandCollapseAction(this.parentNode);
+                }
+            );
+
+            let expanderElement = $actionElement.querySelector(".expander");
 
             // setup click to show/hide requests
-            $('.expander', $actionElement).click( function(event) {
-                expandCollapseAction(this.parentNode);
-            })
+            expanderElement.addEventListener(
+                "click",
+                function () {
+                    expandCollapseAction(this.parentNode);
+                }
+            );
+
             // setup ondblclick to do nothing since a dblclick causes the following event sequence to be dispatched:
             // dblclick ::= click -> click -> dblclick
-            .dblclick( function(event) {
-                event.stopPropagation();
-            });
+            expanderElement.addEventListener(
+                "dblclick",
+                function (event) {
+                    event.stopPropagation();
+                }
+            );
 
             // insert into DOM
-            $actions.append($actionElement);
+            $actions.appendChild($actionElement)
 
             // preprocess action's requests
             preprocessRequests(action.requests);
         }
 
         // insert the actions into the DOM
-        $actionlist.append($actions);
+        actionlist.appendChild($actions);
 
         // show them
-        $actions.slideDown(200);
+        show($actions);
 
         // test parameters and results
-        populateKeyValueTable($valueLog, transaction.valueLog);
+        populateKeyValueTable(valueLog, transactionData.valueLog);
     }
 
-    // the on load setup
-    $(document).ready( function() {
+    window.addEventListener("DOMContentLoaded", function () {
         init();
 
-        var $progress = $('#progressmeter');
+        const progress = getElementById("progressmeter");
 
         try {
-            $progress.show(100);
+            show(progress);
 
             loadJSON();
 
-            // take care of the size of the content display area to
-            // adjust it to the window size
-            $(window).bind("resize", function(event) {
-                resizeContent();
-            });
-
             // setup onclick for the tabbed panel in the request content
             // area
-            $('.tabs-nav li', $requestContent).click( function(event) {
-                activateTab(this);
-            });
-
-            $('#wrapper').splitter({ orientation: 'horizontal', limit: 150 });
-
-            // resize in the beginning already
-            resizeContent();
+            requestContent.querySelectorAll(".tabs-nav li")
+                .forEach(li => li.addEventListener("click", function () {
```

Comment:
Does this work? AFAIK, `querySelectorAll` returns a NodeList and no array.

Reply:
Yes it works. See:
https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach

Reply:
Hm... looks like this is not defined in any spec. I'm not sure whether it's good practice to use non-spec'ed APIs without having polyfills as fallback.

Reply:
Now I use a Array.prototype.forEach.call()